runtime/util/optinfo.c

```diff
@@ -673,6 +673,24 @@ getLineNumberForROMClassFromROMMethod(J9JavaVM *vm, J9ROMMethod *romMethod, J9RO
 	return number;
 }
 
+U_32*
+getNumberOfPermittedSubclassesPtr(J9ROMClass *romClass)
+{
+	U_32 *ptr = getSRPPtr(J9ROMCLASS_OPTIONALINFO(romClass), romClass->optionalFlags, J9_ROMCLASS_OPTINFO_PERMITTEDSUBCLASSES_ATTRIBUTE);
+
+	Assert_VMUtil_true(ptr != NULL);
```

Comment:
Will this not be NULL for unsealed classes? Likely better to NULL check around the SRP get and allow NULL to be returned.

Reply:
The method declaration description indicates that this method is called only for a sealed class.